cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -27,10 +27,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 // CNV-4134 virtctl upload
 
 // CNV-4080 VMI exposed in UI
+* You can view and manage virtual machine instances that are independent of virtual machines within the web console.
```

Comment:
"within the web console." is ambiguous, how about "within the virtual machine lists."

Reply:
Guohua - I made the change you noted. To my best knowledge, we refer to the "web console." During the peer review, this might get changed back to web console if the peer reviewer notes that a reference to web console is more appropriate.. Just mentioning so you know. Thank you.